### PR TITLE
chore: add @[simp] to useful lemmas about Int

### PR DESCRIPTION
**Pull request by semorrison**

### CODE REVIEW

**Comment by semorrison** [comment]
This can wait for CI to complete on https://github.com/leanprover-community/mathlib4/pull/7688, to make sure we don't mess up the linter there.

**Comment by digama0** [comment]
In case you aren't aware, the reason these are not marked as global simp lemmas is because they are subsumed by more general theorems in mathlib. That said I'm not particularly happy about handling the situation in this way because it's less good to users of std without mathlib.

**Comment by semorrison** [comment]
Yes, I assumed as much, but I think that is increasingly less relevant as other consumers of Std appear. Mathlib will just have to cope with the duplication.

File: Std/Data/Int/DivMod.lean [hunk 1]
```diff
@@ -43,7 +43,7 @@ theorem negSucc_ediv (m : Nat) {b : Int} (H : 0 < b) : -[m+1] / b = -(div m b +
   | -[_+1] => rfl
 
 -- Will be generalized to Euclidean domains.
```
Comment (fgdorais) [inline Std/Data/Int/DivMod.lean:45]: Delete comment.

**Review by fgdorais — APPROVED** [review]
Looks great to me!

File: Std/Data/Int/DivMod.lean (hunk 1 above)
Comment (semorrison) [inline Std/Data/Int/DivMod.lean:45]:
```suggestion
```

**Pull request merged**